src/wrapped_radix_engine_toolkit/high_level/ret_api.rs

```diff
@@ -123,14 +123,11 @@ pub fn modify_manifest_lock_fee(
 /// # Panics
 /// Panics if any of the TransactionGuarantee's `instruction_index` is out of
```

Comment:
Please update doc here as well, saying we panic if we are out of bounds, no longer true since replaced with an error, right?

Reply:
Correct. I know I removed this comment, maybe that got lost in rebasing, will remove again